### PR TITLE
AAP-357 added links to builder and navigator guides

### PR DESCRIPTION
**Pull request by emurtoug**
1. modules/dev-guide/con-content-workflows.adoc:

**Under Additional resources**
Added links to [Ansible Navigator Creator Guide](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/%7BPlatformVers%7D/html/ansible_navigator_creator_guide) and [Creating and Consuming Execution Environments](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/%7BPlatformVers%7D/html/creating_and_consuming_execution_environments/index).

2. modules/core/execution-environments.adoc:

**Under Additional resources**
Added link to [Creating and Consuming Execution Environments](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/%7BPlatformVers%7D/html/creating_and_consuming_execution_environments/index).

### CODE REVIEW

File: downstream/modules/dev-guide/con-content-workflows.adoc [hunk 1]
```diff
@@ -15,6 +15,6 @@ As {ControllerName} shifts to using {ExecEnvName}, tools like {Navigator} and {B
 [role="_additional-resources"]
 .Additional resources
 
-* See the link:[Ansible Navigator Creator Guide] for more on using {Navigator}.
-* For more information on {Builder}, see the link:[Ansible Builder Creator Guide].
+* See the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/%7BPlatformVers%7D/html/ansible_navigator_creator_guide[Ansible Navigator Creator Guide] for more on using {Navigator}.
+* For more information on {Builder}, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/%7BPlatformVers%7D/html/creating_and_consuming_execution_environments/index[Creating and Consuming Execution Environments].
```
[the remark this answers is not in the document]
Comment (ariordan-redhat) [reply downstream/modules/dev-guide/con-content-workflows.adoc:19]: Suggestion: remove the `////` comment lines before and after the additional resources content

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Looks like the `{` and `}` characters were rendered weirdly.

File: downstream/modules/dev-guide/con-content-workflows.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (emurtoug) [reply downstream/modules/dev-guide/con-content-workflows.adoc]: Thanks, Aine! That's done now.

File: downstream/modules/core/con-execution-environments.adoc [hunk 2]
```diff
@@ -33,4 +33,5 @@ You can define and create an automation execution environment using {Builder}.
 * Currently, modules cannot include xrefs, so you cannot include links to other content in your collection. If you need to link to another assembly, add the xref to the assembly that includes this module.
 * For more details on writing concept modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
 * Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
+* For more information on {Builder}, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/%7BPlatformVers%7D/html/creating_and_consuming_execution_environments/index[Creating and Consuming Execution Environments].
 ////
```
[the remark this answers is not in the document]
Comment (emurtoug) [reply downstream/modules/core/con-execution-environments.adoc]: Thanks! I've fixed that issue with the { } characters now.

File: downstream/modules/core/con-execution-environments.adoc [hunk 3]
```diff
@@ -33,5 +32,4 @@ You can define and create an automation execution environment using {Builder}.
 * Currently, modules cannot include xrefs, so you cannot include links to other content in your collection. If you need to link to another assembly, add the xref to the assembly that includes this module.
 * For more details on writing concept modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
 * Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
```
Comment (ariordan-redhat) [inline downstream/modules/core/con-execution-environments.adoc:34]:
```suggestion
```

**Review by ariordan-redhat — APPROVED** [review]
Doc built OK and links are valid.
Looks good to me.

**Pull request merged**